BaseClasses.py

```diff
@@ -259,7 +266,7 @@ def set_item_links(self):
                         "link_replacement": replacement_prio.index(item_link["link_replacement"]),
                     }
 
-        for name, item_link in item_links.items():
+        for _name, item_link in item_links.items():
```

Comment:
if name is unused, why not discard it?

Reply:
I considered `item_links.values()` to get rid of it. But in this PR I had a goal of minimizing possible disruption.

The higher the chance of me messing something up, the less likely I am to include it in this PR.